tests/components/sleepiq/__init__.py

```diff
@@ -1 +1,55 @@
 """Tests for the sleepiq component."""
+
+from unittest.mock import Mock
+
+from homeassistant.components.sleepiq.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_SCAN_INTERVAL, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+async def init_integration(
```

Comment:
You can make both of these fixtures instead and then return the config entry. Fixtures can inherit other fixtures as well, so you do not need to call helper methods at the beginning of your tests to set stuff up.

Docs: https://docs.pytest.org/en/latest/how-to/fixtures.html
Example: https://github.com/home-assistant/core/blob/dev/tests/components/unifiprotect/conftest.py#L148

Reply:
Thanks, your suggestion cleaned things up a bit, especially since I was able to parametrize the mock requests for single vs double bed.